unit/testing-utils/load_java_class.cpp

```diff
@@ -103,6 +107,12 @@ symbol_tablet load_java_class(
   const typet &class_type=class_symbol.type;
   REQUIRE(class_type.id()==ID_struct);
 
+#ifdef CATCH_PLATFORM_LINUX
+  char temp [ PATH_MAX ];
+  std::string path = getcwd(temp,PATH_MAX) ? std::string(temp) : "";
+  INFO("Working directory: " << path);
+#endif
```

Comment:
What about `get_current_working_directory` (from util/file_util.h)?

Reply:
Ah wonderful - someones done the work for me - thanks!